src/Illuminate/Support/Str.php

```diff
@@ -2014,4 +2016,28 @@ public static function flushCache()
         static::$camelCache = [];
         static::$studlyCache = [];
     }
+
+    /**
+     * Sanitize the given string.
+     *
+     *
+     * See: https://symfony.com/doc/current/html_sanitizer.html
+     *
+     * @param  string  $string  The input string to sanitize.
+     * @param  HtmlSanitizerConfig|null  $config  Custom configuration to use for sanitizing.
+     * @return string The sanitized string.
+     */
+    public static function sanitize($string, ?HtmlSanitizerConfig $config = null)
+    {
+        if (is_null($string)) {
+            return null;
+        }
+
+        if (! $config) {
+            $config = (new HtmlSanitizerConfig)
+                ->allowSafeElements();
+        }
```

Comment:
There's no need to use the `not` operator for a falsy value here (`if ($config === null)` would have sufficed):
```suggestion
        $config ??= (new HtmlSanitizerConfig)
            ->allowSafeElements();
```